src/Mailgun/Api/Pagination.php

```diff
@@ -10,6 +10,7 @@
 namespace Mailgun\Api;
 
 use Mailgun\Assert;
+use Mailgun\Resource\Api\mixed;
```

Comment:
Plan B is to set a generic Interface that should be implemented by all possible objects that we are going to expect.